README.md

```diff
@@ -604,11 +654,12 @@ The mapping from Http Json Response to SQL table schema is done via Flink's Json
   - Changed API for public HttpSink builder. The `setHttpPostRequestCallback` expects a `PostRequestCallback`
     of generic type [HttpRequest](src/main/java/com/getindata/connectors/http/internal/sink/httpclient/HttpRequest.java)
     instead `HttpSinkRequestEntry`.
+- Version 0.20
```

Comment:
I think this should be mentioned in changelog. Readme should reflect current state. What do you think?